### DISCUSSION TITLE
Pricing Realtime API vs STT-TTS

### DISCUSSION THREAD

**Discussion by Laki4Ever79** [microsoft/call-center-ai#326] [Q&A] [answered]
Hello @clemlesne, I really appreciate your work. Could you maybe provide some rough estimation about potential cost (per call, per token or other) comparison between STT-TTS approach which was present in the v11.1.0 versions and older to the current approach which takes advantage of Realtime API? Or even better how much entire implementation through Azure services could cost?

Kind regards,
Aleksa

**Comment by clemlesne** [comment, marked as answer]
The realtime API is not yet integrated.

I didn’t yet made any estimation. But the Azure OpenAI realtime API is way pricier, the total cost per hour of conversation should raise a bit.

You can follow the work in the dedicated issue: https://github.com/microsoft/call-center-ai/issues/210

Plus, I created an issue to create a first public cost estimation: https://github.com/microsoft/call-center-ai/issues/327

**Reply by clemlesne** [reply]
Costs breakdown added in 1c80093357f63a24aeec8857313c383a3130e964.
